mirage/factories/institution.ts

```diff
@@ -35,17 +35,33 @@ export default Factory.extend<Institution & InstitutionTraits>({
         afterCreate(institution, server) {
             const userMetrics = server.createList('institution-user', 15);
             const departmentNames = ['Architecture', 'Biology', 'Psychology'];
+
             const departmentMetrics = departmentNames.map(name =>
                 server.create('institution-department', { name }));
+
             const userCount = userMetrics.length;
             let publicProjectCount = 0;
             let privateProjectCount = 0;
             userMetrics.forEach(({ publicProjects, privateProjects }) => {
                 publicProjectCount += publicProjects;
                 privateProjectCount += privateProjects;
             });
+
             const summaryMetrics = server.create('institution-summary-metric', { id: institution.id });
-            summaryMetrics.update({ publicProjectCount, privateProjectCount, userCount });
+
+            summaryMetrics.update({
+                publicProjectCount,
+                privateProjectCount,
+                userCount,
+                publicRegistrationCount: 1250,
+                preprintCount: 11250,
+                embargoedRegistrationCount: 456,
+                storageByteCount: 47382032,
+                publicFileCount: 87,
+                monthlyLoggedInUserCount: 24563,
+                monthlyActiveUserCount:456,
+            });
```

Comment:
It seems like the logic that was here before this PR was trying to be smart and infer `publicProjectCount`, `privateProjectCount` and `userCount` from the 15 `institution-users` created on line 36 to make the data look a bit more "realistic". While I think that was likely a good idea when we showed the user numbers and summary numbers on the same page before, I don't think this entire `summaryMetrics.update` is necessary anymore.

Would you mind removing this `update` block and lines 42-48 (`const userCount` through `userMetrics.forEach`)?